### PR TITLE
Feature/4012 forgot password base screen

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-4012

#### Changes proposed in this Pull Request:

- Added forgot password screen base

#### Screenshots / Screen Recording (if applicable)

https://github.com/etn-ccis/blui-react-workflows/assets/120575281/9f808e6c-e7f7-42f0-881d-ddfff9133a81

#### To Test:

- yarn start:example2

#### Any specific feedback you are looking for?

- I will add the test cases in the next PR.

### CODE REVIEW

File: login-workflow/src/new-architecture/components/WorkflowCard/WorkflowCard.types.ts [hunk 1]
```diff
@@ -14,8 +15,8 @@ export type WorkflowCardBaseProps = BoxProps & {
 
 export type WorkflowCardHeaderProps = CardHeaderProps;
 
-export type WorkflowCardInstructionProps = Omit<TypographyProps, 'children'> & {
-    instructions?: string;
+export type WorkflowCardInstructionProps = TypographyProps & {
+    instructions?: string | React.ReactNode;
```
Comment (ektaghag-eaton) [inline login-workflow/src/new-architecture/components/WorkflowCard/WorkflowCard.types.ts:19]: Will it accept JSX element?

**Comment by ektaghag-eaton** [comment]
Hi @surajeaton , changes looks good. Few doubts I have related to type. Please check my comments. Thanks!!

**Comment by ektaghag-eaton** [comment]
Getting console error
Screenshot 2023-06-02 at 12 46 15 PM

**Comment by surajeaton** [comment]
@ektaghag-eaton Updated. Please review

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**